crates/biome_cli/src/commands/check.rs

```diff
@@ -74,13 +76,37 @@ pub(crate) fn check(
         session.app.console,
         cli_options.verbose,
     )?;
+    let (editorconfig, editorconfig_diagnostics) = {
+        let search_path = loaded_configuration
+            .directory_path
+            .clone()
+            .unwrap_or_else(|| {
+                let fs = &session.app.fs;
+                fs.working_directory().unwrap_or_default()
+            });
+        load_editorconfig(&session.app.fs, search_path)?
+    };
+    for diagnostic in editorconfig_diagnostics {
+        session.app.console.error(markup! {
+            {PrintDiagnostic::simple(&diagnostic)}
+        })
+    }
```

Comment:
```suggestion
    let fs = &session.app.fs;
    let (editorconfig, editorconfig_diagnostics) = {
        let search_path = loaded_configuration
            .directory_path
            .clone()
            .unwrap_or_else(|| {
                fs.working_directory().unwrap_or_default()
            });
        load_editorconfig(fs, search_path)?
    };
    for diagnostic in editorconfig_diagnostics {
        session.app.console.error(markup! {
            {PrintDiagnostic::simple(&diagnostic)}
        })
    }
```

That's what I was trying to suggest. Sorry if I wasn't clear enough